### PR TITLE
Fixes Enemy AoE Spells to Affect the Caster

### PR DESCRIPTION
**Pull request by Steven-Biro**
Overwrites the code that checks and skips casting if the target is the caster

### CODE REVIEW

File: FF1Lib/Hacks.cs [hunk 1]
```diff
@@ -15,72 +15,72 @@ public partial class FF1Rom : NesRom
 		public const int PartyShuffleSize = 3;
```
Comment (Steven-Biro) [inline FF1Lib/Hacks.cs]: Accidentally added this file and it isn't needed, unsure how to remove it, the updates are all line endings

**Comment by Steven-Biro** [comment]
This patch overwrites the following:

    LDA $6BCE                   ; BUGGED - see if this enemy is the attacker, and skip it if it is.
    CMP btl_attacker            ;  This doesn't make any sense, as there's no reason why "target all" spells
    BEQ @Next                   ;  should skip the caster.  It doesn't behave this way for players.

File: FF1Lib/BugFixes.cs [hunk 2]
```diff
@@ -73,5 +73,11 @@ public void FixEnemyStatusAttackBug()
 		{
 			Put(0x32812, Blob.FromHex("DF")); // This is the craziest freaking patch ever, man.
 		}
+
+		public void FixEnmeyAOESpells()
```
Comment (latzio) [inline FF1Lib/BugFixes.cs]: Gotta typo here in Enemy

File: FF1Lib/FF1Rom.cs [hunk 3]
```diff
@@ -108,6 +108,11 @@ public void Randomize(Blob seed, Flags flags)
 				FixSpellBugs();
 			}
 
+			if (flags.EnemySpellsTargetingAllies)
+			{
+				FixEnmeyAOESpells();
```
Comment (latzio) [inline FF1Lib/FF1Rom.cs]: Typo here too naturally.

File: FF1Lib/FF1Rom.cs [hunk 4]
```diff
@@ -86,17 +86,17 @@ public void Randomize(Blob seed, Flags flags)
 			var rng = new MT19337(BitConverter.ToUInt32(seed, 0));
 
 			UpgradeToMMC3();
-			EasterEggs();
```
Comment (latzio) [inline FF1Lib/FF1Rom.cs]: There are a lot of unnecessary whitespace changes in this file... here and several other places.

File: FF1Lib/Flags.cs [hunk 5]
```diff
@@ -1,8 +1,8 @@
-﻿using System;
+﻿using System;
```
Comment (latzio) [inline FF1Lib/Flags.cs]: Something is really up with all the whitespace changes your editor is making.

**Comment by Steven-Biro** [comment]
Fixed the problems  @latzio  pointed out

**Comment by Steven-Biro** [comment]
Solves issue #105

**Comment by Steven-Biro** [comment]
Included in #120
